src/main/java/com/volmit/iris/core/decrees/DecIris.java

```diff
@@ -127,18 +131,31 @@ public void bitwise(
             sender().sendMessage(C.RED + "The operator you entered: (" + operator + ") is invalid!");
             return;
         }
-        sender().sendMessage(C.GREEN + "" + value1 + " " + C.GREEN + operator.replaceAll("<", "≺").replaceAll(">", "≻") + " " + C.GREEN + value2 + C.GREEN + " returns " + C.GREEN + v);
+        sender().sendMessage(C.GREEN + "" + value1 + " " + C.GREEN + operator.replaceAll("<", "≺").replaceAll(">", "≻").replaceAll("%", "％") + " " + C.GREEN + value2 + C.GREEN + " returns " + C.GREEN + v);
     }
 
     @Decree(description = "Toggle debug")
     public void debug(
-            @Param(name = "on", description = "Whether or not debug should be on", defaultValue = "true")
-                    boolean on
+            @Param(name = "on", description = "Whether or not debug should be on", defaultValue = "toggle")
+                    String on
```

Comment:
You can instead use a capital B `Boolean` which CAN be null, offer new default, then change the name to "enabled" and do a null-check. If "enabled" is null, toggle it, otherwise set debug to "enabled variable"

Reply:
That's indeed better. On it.